src/traces/histogram/calc.js

```diff
@@ -171,15 +171,19 @@ module.exports = function calc(gd, trace) {
             var cdi = {
                 p: pos[i],
                 s: size[i],
-                b: 0,
-                pts: inputPoints[i]
+                b: 0
             };
-            if(uniqueValsPerBin) {
-                cdi.p0 = cdi.p1 = (inputPoints[i].length) ? pos0[inputPoints[i][0]] : pos[i];
-            }
-            else {
-                cdi.p0 = roundFn(binEdges[i]);
-                cdi.p1 = roundFn(binEdges[i + 1], true);
+
+            // pts and p0/p1 don't seem to make much sense for cumulative distributions
+            if(!cumulativeSpec.enabled) {
```

Comment:
A questionable decision... this gets at some of the same issues that led to `cumulative.currentbin` but in principle a CDF shows the sum of all the data prior to a specific point, not the data within that bin. perhaps I should shift `p0` and `p1` depending on `currentbin` so that the hover label at least gets the description "all the data prior to X" precisely correct, even though it'll be a different value than the center of the bar...

And then what about `pts`? You could say it should be included and again should be "all the data prior to X" but then it would be meaningless to select a single bar and not all the bars before it. Alternatively `pts` could contain "all the data that was added in this bar" which might be more useful in terms of selecting a single bar, but it's not really what that bar means.

Reply:
> so that the hover label at least gets the description "all the data prior to X"

Yeah. I think something along those lines would be useful down the road. 

> You could say it should be included and again should be "all the data prior to X" but then it would be meaningless to select a single bar and not all the bars before it. Alternatively pts could contain "all the data that was added in this bar" 

Sounds to me like both _all-data-prior-to-X_ and _data-added-in-this-bar_ point lists could be useful, so we'll might have to emit both in the future. Leaving this discussion for another time is :ok_hand: with me. I doubt that most plotly.js users even know about cumulative histograms.

Reply:
Great, I'll make a new issue  to discuss how this should work for CDFs.